clang/tools/clang-offload-extract/ClangOffloadExtract.cpp

```diff
@@ -31,27 +30,39 @@
 using namespace llvm;
 using namespace llvm::object;
 
-static cl::opt<bool> Help("h", cl::desc("Alias for -help"), cl::Hidden);
```

Comment:
Is this intended to be removed? I think the test fail is due to this.
Please take a look.
Thanks

Reply:
Indeed, the `-h` option is built-in to the LLVM Command Line Parser. Trying to create an overwrite will mess up the results of the help. Since the overwrite was trying to implement the same functionality as what the built-in does, it should be removed.

Reply:
In this case, I'm pretty sure the test needs to be updated, let me see if I can determine how.